internal/engine/build_and_deployer.go

```diff
@@ -74,8 +74,9 @@ func (composite *CompositeBuildAndDeployer) BuildAndDeploy(ctx context.Context,
 		}
 
 		if redirectErr, ok := err.(buildcontrol.RedirectToNextBuilder); ok {
-			s := fmt.Sprintf("falling back to next update method because: %v\n", err)
-			logger.Get(ctx).Write(redirectErr.Level, []byte(s))
+			l := logger.Get(ctx).WithFields(logger.Fields{logger.FieldNameBuildEvent: "1"})
+			s := fmt.Sprintf("Live Update temporarily disabled…\nREASON: %v\n", err)
```

Comment:
this isn't quite accurate. This codepath handles all redirects, not just live update.

would it make sense to create a new error type (LiveUpdateFailed?), or to add this error message there?

Reply:
Thanks for catching this! For now, I reverted the error message to the more general "Falling back" notice. I agree that we should have a distinct error type for LiveUpdate. But in the interest of landing this PR more quickly, I will do that in a separate PR.

Reply:
hmmm...i don't see an update here? did you forget to push?